source/clear-linux/get-started/bare-metal-install-desktop/bare-metal-install-desktop.rst

```diff
@@ -66,6 +66,53 @@ these steps.
 
 .. _install-on-target-end:
 
+Confirm network connection
+==========================
+
+Confirm that there is a wired connection before launching the installer.
```

Comment:
I would not go so far as to call out a Wired versus Wireless connection.
They certainly can just be configured wireless, many laptops won't even have wired options without adding a network dongle.

Otherwise, it seems good to me.

Reply:
Got it. I will revise accordingly. Don't worry on the specific "square icon". I added a new Figure for users. ( I perused a lot of GNOME 3 top bar docs, but couldn't find a definitive answer.)

Reply:
Yes, my searching of Gnome documents also found nothing for that specific name of the that icon grouping. :(